src/zentables/__init__.py

```diff
@@ -13,6 +13,6 @@
     df.zen.pretty()
 """
 
-from .zentables import ZenTablesAccessor, set_options
+from .zentables import ZenTablesAccessor, _do_suppression, set_options
```

Comment:
It is a little odd to indicate a function is private (i.e., it is named `_name` instead of `name`) but also export it as part of `from zentables import *`

Reply:
I am struggling to export it as part of *.